Marlin/src/gcode/lcd/M0_M1.cpp

```diff
@@ -73,9 +73,9 @@ void GcodeSuite::M0_M1() {
       ExtUI::onUserConfirmRequired(GET_TEXT_F(MSG_USERWAIT));
   #elif ENABLED(DWIN_LCD_PROUI)
     if (parser.string_arg)
-      dwinPopupConfirm(ICON_BLTouch, parser.string_arg, GET_TEXT_F(MSG_USERWAIT));
+      dwinPopupContinue(ICON_BLTouch, F(parser.string_arg), GET_TEXT_F(MSG_USERWAIT));
```

Comment:
C strings, `PROGMEM` strings, and `F` strings all need to be treated differently. If `dwinPopupContinue` wants to do something with a C string then there must a separate `dwinPopupContinue` method that takes a C string. If we wanted a method that accepts a `PROGMEM` string (`PGM_P`) it must be named `dwinPopupContinue_P` because `PROGMEM` strings cannot be distinguished from regular C strings. `PROGMEM` strings can be wrapped in `FPSTR()` to be passed to a method taking an `F` string. And `F` strings can be wrapped in `FTOP()` to be passed to a method taking a `PROGMEM` string.